samtranslator/internal/schema_source/aws_serverless_graphqlapi.py

```diff
@@ -14,8 +14,39 @@
 
 
 # TODO: add docs
+class LambdaAuthorizer(BaseModel):
+    AuthorizerResultTtlInSeconds: Optional[PassThroughProp]
+    AuthorizerUri: Optional[PassThroughProp]
+    IdentityValidationExpression: Optional[PassThroughProp]
+
+
+class OpenIDConnect(BaseModel):
+    AuthTTL: Optional[PassThroughProp]
+    ClientId: Optional[PassThroughProp]
+    IatTTL: Optional[PassThroughProp]
+    Issuer: Optional[PassThroughProp]
+
+
+class UserPool(BaseModel):
+    AppIdClientRegex: Optional[PassThroughProp]
+    AwsRegion: Optional[PassThroughProp]
+    DefaultAction: Optional[PassThroughProp]
+    UserPoolId: Optional[PassThroughProp]
+
+
+class AdditionalAuth(BaseModel):
+    Type: str
```

Comment:
Literal?

Reply:
True, let me make a type for this instead.